### PR TITLE
 🌱Add log to provide more info in case error happen

### PR DESCRIPTION
**Pull request by jichenjc**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**TODOs**:

- [ ] squashed commits
- if necessary:
  - [ ] includes documentation
  - [ ] adds unit tests

/hold

### CODE REVIEW

File: pkg/cloud/services/loadbalancer/loadbalancer.go [hunk 1]
```diff
@@ -248,6 +248,7 @@ func (s *Service) getOrUpdateAllowedCIDRS(openStackCluster *infrav1.OpenStackClu
 	listener.AllowedCIDRs = capostrings.Unique(listener.AllowedCIDRs)
 
 	if !reflect.DeepEqual(allowedCIDRs, listener.AllowedCIDRs) {
+		s.scope.Logger.Info("CIDRs not match, update listern", "expected CIDRs", allowedCIDRs, "load balancer existing CIDR", listener.AllowedCIDRs)
```
Comment (lentzi90) [inline pkg/cloud/services/loadbalancer/loadbalancer.go:251]: Nit:
```suggestion
		s.scope.Logger.Info("CIDRs do not match, update listener", "expected CIDRs", allowedCIDRs, "load balancer existing CIDR", listener.AllowedCIDRs)
```

File: pkg/cloud/services/loadbalancer/loadbalancer.go [hunk 2]
```diff
@@ -248,6 +248,7 @@ func (s *Service) getOrUpdateAllowedCIDRS(openStackCluster *infrav1.OpenStackClu
 	listener.AllowedCIDRs = capostrings.Unique(listener.AllowedCIDRs)
 
 	if !reflect.DeepEqual(allowedCIDRs, listener.AllowedCIDRs) {
+		s.scope.Logger.Info("CIDRs do not match, start to update lister", "expected CIDRs", allowedCIDRs, "load balancer existing CIDR", listener.AllowedCIDRs)
```
Comment (tobiasgiese) [inline pkg/cloud/services/loadbalancer/loadbalancer.go:251]:
```suggestion
		s.scope.Logger.Info("CIDRs do not match, start to update listener", "expected CIDRs", allowedCIDRs, "load balancer existing CIDR", listener.AllowedCIDRs)
```

There's still a typo @jichenjc :)

Comment (jichenjc) [reply pkg/cloud/services/loadbalancer/loadbalancer.go:251]: :( 

will update ..

**Review by lentzi90 — COMMENTED** [review]
/lgtm

**Pull request merged**